README.markdown

```diff
@@ -443,6 +443,35 @@ Keep this in mind when writing your jobs: you may want to throw
 exceptions you would not normally throw in order to assist debugging.
 
 
+#### Rails example
+
+If you are using ActiveJob here's how your job definition will look:
+
+``` ruby
+class ArchiveJob < ApplicationJob
+  queue_as :file_serve
+
+  def perform(repo_id, branch = 'master')
+    repo = Repository.find(repo_id)
+    repo.create_archive(branch)
+  end
+end
+```
+
+``` ruby
+class Repository
+  def async_create_archive(branch)
+    ArchiveJob.perform_later(self.id, branch)
+  end
+end
+```
+
+It is important to run `ArchiveJob.perform_later(self.id, branch)` rather than `Resque.enqueue(Archive, self.id, branch)`.
+Otherwise the Resque will process the job without actually doing anything.
```

Comment:
I would remove the word `the` from between `Otherwise` and `Resque` to improve the grammar a bit.